package.json

```diff
@@ -91,6 +98,7 @@
   "scripts": {
     "lint": "eslint .",
     "test": "mocha --require test/support/env --reporter spec --bail --check-leaks test/ test/acceptance/",
+    "start": "node index.js",
```

Comment:
```suggestion
```

I assume that we don't need this change, as this package file is globally for the library.